sass/base/_icons.scss

```diff
@@ -0,0 +1,66 @@
+%icon-font-base {
+  font-family: 'AppBox';
+  speak: none;
+  font-style: normal;
+  font-weight: normal;
+  font-variant: normal;
+  text-transform: none;
+  line-height: 1;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+}
+
+$iconPrimary:
+(computer 2em 2.5rem "\e600")
+(cog 2em 2.5rem "\e611")
+(lab 2em 2.5rem "\e612")
+(airplane 2em 2.5rem "\e613")
+(screen 1.25em 1.25rem "\e601")
+(apple 1.25em 1.25rem "\e607")
+(flag 1.25em 1.25rem "\e608")
+(coffee 1.25em 1.25rem "\e609")
+(cubes 1.25em 1.25rem "\e60a")
+(phone 1.25em 1.25rem "\e60c")
+(envelope 1.25em 1.25rem "\e60d")
+(location 1.25em 1.25rem "\e60e")
+(bubble 1.25em 1.25rem "\e60f");
+
+$iconBase:
+(dribbble 1.25em "\e602")
+(google-plus 1.25em "\e603")
+(twitter 1.25em "\e604")
+(facebook 1.25em "\e605")
+(pinterest 1.25em "\e606")
+(search 1em "\e610")
+(play4 1.25em "\e614")
+(behance 1.25em "\e60b")
+(apple2 1.25em "\e615")
+(android 1.25em "\e616")
+(quote 2em "\e617");
+
+@each $iconP in $iconPrimary {
+  .icon-#{nth($iconP, 1)} {
+    @extend %icon-font-base;
+    font-size: #{nth($iconP, 2)};
+    background-color: transparent;
+    color: color('accent');
+    width: 100%;
+    padding: nth($iconP, 3);
+    @include icon-border($color: border('primary'));
```

Comment:
Just a code style nit: place all `@extend`s and `@include`s at the top of the block, followed by an empty line and then the literal styles.

```
.icon-#{nth($iconP, 1)} {
   @extend %icon-font-base;
   @include icon-border($color: border('primary'));

   font-size: #{nth($iconP, 2)};
   background-color: transparent;
   color: color('accent');
   width: 100%;
   padding: nth($iconP, 3);
}
```